doc/en/user/source/community/ogc-api/styles.rst

```diff
@@ -0,0 +1,4 @@
+OGC API Styles
+==============
+
+A `OGC Styles API <https://github.com/opengeospatial/ogcapi-styles>`_ based on the early draft of this specification, allowing to describe, retrieve and update styles present on the server. Styles are linked back and forwards with their associated collections in both the Features and Tiles API
```

Comment:
"allowing to" seems a bit strange. Maybe:

"An `OGC Styles API <https://github.com/opengeospatial/ogcapi-styles>`_ implementation based on an early draft of this specification, that supports describing, retrieving and updating styles present on the server. Styles are linked back and forwards with their associated collections in both the Features and Tiles APIs."